### PR TITLE
make webtest pip installable

### PR DESCRIPTION
**Pull request by joshmoore**
Follows the pattern in https://github.com/openmicroscopy/weberror/pull/7

Note: flake8 added to requirements.txt since the file is otherwise empty which leads to an error.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+python: 2.7
 
 # This (sudo: false) is needed to "run on container-based infrastructure" on
 # which cache: is available
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
 
-before_install: pip install --user flake8
+before_install:
+  - pip install flake8
```
Comment (sbesson) [inline .travis.yml]: `pip install -r requirements.txt` then?

**Comment by joshmoore** [comment]
Self-merging to test deployment to PyPI.

**Comment by joshmoore** [comment]
Note: saw this during registration:

```
warning: check: Possible incomplete section title.
Treating the overline as ordinary text because it's so short. (line 37)

warning: check: Blank line missing before literal block (after the "::")? Interpreted as a definition list item. (line 40)

warning: check: Possible incomplete section title.
Treating the overline as ordinary text because it's so short. (line 48)

warning: check: Blank line missing before literal block (after the "::")? Interpreted as a definition list item. (line 51)

Registering omero-webtest to https://pypi.python.org/pypi
Server response (200): OK
```

**Comment by atarkowska** [comment]
sorry but I think that doesn't work ;-)

**Pull request merged**